org.eclipse.jdt.ls.core/META-INF/MANIFEST.MF

```diff
@@ -31,7 +31,8 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.12.0",
  org.eclipse.lsp4j.jsonrpc,
  org.eclipse.xtend.lib,
  org.eclipse.xtext.xbase.lib
-Export-Package: org.eclipse.jdt.ls.core.internal;x-friends:="org.eclipse.jdt.ls.tests",
+Export-Package: org.eclipse.jdt.ls.core,
```

Comment:
I'd rather keep internal references with x-friends. The code is internal to the jdt.ls project so I'm perfectly fine to have a jdt.ls plugin depend on that.
 
There's currently no code in jdt.ls that I'd be willing to publicly expose to the world, until explicitly requested by 3rd party adopters. Even then that'd require careful consideration.

So still -1 on that.